ndo_schemas.tf

```diff
@@ -398,31 +398,33 @@ locals {
     for schema in local.schemas : [
       for template in try(schema.templates, []) : [
         for vrf in try(template.vrfs, []) : {
-          key                    = "${schema.name}/${template.name}/${vrf.name}"
-          schema_id              = mso_schema.schema[schema.name].id
-          template_name          = template.name
-          name                   = "${vrf.name}${local.defaults.ndo.schemas.templates.vrfs.name_suffix}"
-          display_name           = "${vrf.name}${local.defaults.ndo.schemas.templates.vrfs.name_suffix}"
-          layer3_multicast       = try(vrf.l3_multicast, local.defaults.ndo.schemas.templates.vrfs.l3_multicast)
-          vzany                  = try(vrf.vzany, local.defaults.ndo.schemas.templates.vrfs.vzany)
-          ip_data_plane_learning = try(vrf.data_plane_learning, local.defaults.ndo.schemas.templates.vrfs.data_plane_learning) ? "enabled" : "disabled"
-          preferred_group        = try(vrf.preferred_group, local.defaults.ndo.schemas.templates.vrfs.preferred_group)
+          key                           = "${schema.name}/${template.name}/${vrf.name}"
+          schema_id                     = mso_schema.schema[schema.name].id
+          template_name                 = template.name
+          name                          = "${vrf.name}${local.defaults.ndo.schemas.templates.vrfs.name_suffix}"
+          display_name                  = "${vrf.name}${local.defaults.ndo.schemas.templates.vrfs.name_suffix}"
+          layer3_multicast              = try(vrf.l3_multicast, local.defaults.ndo.schemas.templates.vrfs.l3_multicast)
+          vzany                         = try(vrf.vzany, local.defaults.ndo.schemas.templates.vrfs.vzany)
+          ip_data_plane_learning        = try(vrf.data_plane_learning, local.defaults.ndo.schemas.templates.vrfs.data_plane_learning) ? "enabled" : "disabled"
+          preferred_group               = try(vrf.preferred_group, local.defaults.ndo.schemas.templates.vrfs.preferred_group)
+          site_aware_policy_enforcement = try(vrf.site_aware_policy_enforcement, null)
```

Comment:
why without default value?

Reply:
If we push a value without the user specifying it, this will break with older versions that didn't have this option. For versions that do support it, `null` will use the NDO default.